wiki/UI-component/Video.md

```diff
@@ -31,14 +31,75 @@ The [[Video]] component utilizes the embeddable Kaltura player <sup>[[r2](#resou
 <details open="true">
   <summary><strong>Resources</strong></summary><br />
 
-- r1: [Box folder](https://ibm.ent.box.com/folder/108837570522)
+- r1: [Box folder](https://ibm.box.com/s/9q7rdj73m32hufcw1xwl1td6nr88rghm)
 - r2: [Kaltura player](http://player.kaltura.com/docs/api)
 
 <br />
 </details>
 
 <hr width="40%" />
 
-<br />
+## Functions
+
+| Variants | Interaction | Functions |
+| -------- | ----------- | --------- |
+| Default | Clicking on `Poster image` | Poster image is replaced with Kaltura video player and video will begin playing. |  
+| with lightbox media viewer | Clicking on `Poster image` | Open LightBoxMediaViewer and video will begin playing. |
+
+
+## Logics
+
+### Default variant
+
+Video allows custom override of all content returned by the [[Video service]] except video duration.
+
+| ID  | Field |
+| --- | ----- |
+| 1   | Caption |
+| 2   | Poster image |
+| 3   | Poster image alt tag |
+
+##### 1. Caption
```

Comment:
I try to follow the data model if possible. I would call this out as copy personally.